source/base/ReferenceCounter.ooc

```diff
@@ -22,6 +22,7 @@ ReferenceCounter: class {
 			}
 		}
 	}
+	count ::= this _count
```

Comment:
Ok, I guess because the value is written to explicitly.
For example, this fails the ReferenceCounter test:
```occ
		counter isSafe = true
		expect(counter isSafe, is equal to(true))
		counter isSafe = false
		expect(counter isSafe, is equal to(false))
		counter isSafe = true
		expect(counter isSafe, is equal to(true))
```
to me it looks like a bug in rock.